src/lv_misc/lv_task.c

```diff
@@ -256,13 +256,13 @@ lv_task_t * lv_task_create_basic(void)
  * @param user_data custom parameter
  * @return pointer to the new task
  */
-lv_task_t * lv_task_create(lv_task_cb_t task_cb, uint32_t period, lv_task_prio_t prio, void * user_data)
+lv_task_t * lv_task_create(lv_task_cb_t task_xcb, uint32_t period, lv_task_prio_t prio, void * user_data)
```

Comment:
I believe the parameter name should be `cb` and not `xcb` for a reason (the MicroPython binding uses it to generate the appropriate wrapper).

Reply:
Thanks, will fix asap. What about lv_async_call(lv_async_cb_t async_xcb, void * user_data).
Just noticed in lv_misc\lv_async.h that comment has "@param task_xcb" and thought to correct that too: but now the question is should be "async_xcb" or "async_cb".
Thanks again, and very happy to learn your great library!